common/test/acceptance/pages/lms/discussion.py

```diff
@@ -380,14 +380,17 @@ def get_selected_sort_preference(self):
         """
         Return the text of option that is selected for sorting.
         """
-        options = self.q(css="body.discussion .forum-nav-sort-control option")
-        return options.filter(lambda el: el.is_selected())[0].get_attribute("value")
+        selected_value = self.browser.execute_script(
```

Comment:
@raeeschachar Please add a comment to that effect, so that other people who come across this pattern understand and also know that it is a workaround and not a usual practice.